### ISSUE TITLE
Show error message when form can't be exported to XML

### ISSUE THREAD

**Issue by lognaturel** [opendatakit/build#180]
It's possible to get into a state where forms can't be exported to XML but no error shows either in the user interface or on the console. I'm experiencing this right now with a form that uses range with the different appearances and I have no idea where the problem is. It would be helpful to get some kind of feedback when hat happens.

**Comment by clint-tseng** [comment]
I'm not sure what this ticket's action item is; it seems we should be filing bugs when problems occur and then we can chase each one down?

**Comment by lognaturel** [comment]
Instead of failing silently, do you have any information at all that can be shown to the user when the XML can't be generated?

**Comment by clint-tseng** [comment]
Not really. Even a stack trace doesn't contain enough information for me to trace through oftentimes.

**Comment by clint-tseng** [comment]
Though I'm not sure why you wouldn't be seeing anything in the console. That shouldn't be possible, and sounds very fishy.
